config/serverless.yml

```diff
@@ -222,11 +222,6 @@ xpack.cases.stack.enabled: false
 # Alerting and action circuit breakers
 xpack.alerting.rules.run.actions.max: 3000
 xpack.alerting.rules.run.timeout: 1m
-xpack.alerting.rules.run.ruleTypeOverrides:
```

Comment:
Question: Removing these wont affect the other projectTypes?

Reply:
To @kc13greiner's point, I think in the long run it might make sense to update the way this parameter is resolved, so that we merges the values from both `serverless.yml` global level and `serverless.(security|observability|search).yml` solution-specific level.